examples/src/bin/gio_futures_await.rs

```diff
@@ -1,4 +1,5 @@
-use gio::prelude::*;
+use gtk::prelude::*;
```

Comment:
Same here.

Reply:
there's no comment to have an empty line between :p